### PR TITLE
PROD-3204 IAC for Stack -> PROD-3172_init-repo

### PR DESCRIPTION
**Pull request by brooketopcoder**
This commit adds all the support for creating the S3 bucket and Cloudfront instance to host the JS file.

### CODE REVIEW

File: iac/uni-nav.deploy.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            HostedZoneId: Z2CIRG3R0ZSGFQ
+            RoleArn: arn:aws:iam::811668436784:role/learning-paths-v5-container-role
+
+Resources:
+
+    Store:
+        Type: AWS::S3::Bucket
+        DeletionPolicy: Delete
+        Properties:
+            AccessControl: PublicRead
```
Comment (testflyjets) [inline iac/uni-nav.deploy.yml:41]: We need to find a solution for hosting the file in S3 that doesn't require the bucket to be open for Public Read. Can you look into best practices for hosting web assets that solve that, possibly using CloudFront and an access origin restriction?

File: iac/deploy-stack.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (testflyjets) [inline iac/deploy-stack.sh:1]: Have you checked to see if most of what this script provides can be done with serverless.com out of the box?

**Review by testflyjets — COMMENTED** [review]
We need to find a solution for the S3 Public Read access, as I just learned this week Wipro considers this a critical security vulnerability unless there's no other way to get the same results. Let's see if we can configure CloudFront to connect securely to S3.

File: iac/deploy-stack.sh (hunk 2 above)
Comment (brooketopcoder) [reply iac/deploy-stack.sh:1]: I didn't find anything last time I checked but will check again now that I'm more familiar.

**Comment by brooketopcoder** [comment]
> We need to find a solution for the S3 Public Read access, as I just learned this week Wipro considers this a critical security vulnerability unless there's no other way to get the same results. Let's see if we can configure CloudFront to connect securely to S3.

Gotcha. Will do.

File: iac/deploy-stack.sh (hunk 2 above)
Comment (brooketopcoder) [reply iac/deploy-stack.sh:1]: I have a hard time finding anything related to serverless specifically. Everything I've found relates specifically to lamdba functions, and this stack doesn't have any lambdas.

**Review by testflyjets — APPROVED** [review]

**Pull request merged**